### DISCUSSION TITLE
Item NBT localized

### DISCUSSION THREAD

**Discussion by LeaPhant** [HypixelDev/PublicAPI#290] [Q&A]
The SkyBlock API returns item NBT localized based on the player's selected language. This leads to issues since sometimes the NBT is the only place to extract required information from.

**Comment by puppy0cam** [comment]
This is unlikely to be fixed as it requires additional processing from the API. In this specific situation, I would recommend you use the colour of the item name to determine rarity.

**Comment by nstringham** [comment]
It would be useful if you could specify a language in the request. This would allow community members to make tools that can use by people in any language

**Comment by puppy0cam** [comment]
> It would be useful if you could specify a language in the request. This would allow community members to make tools that can use by people in any language

Unfortunately the API does not process the data, it only retrieves it from the database. This can result in you seeing items that are already localised to different languages.

**Comment by MartinNemi03** [comment]
> 
> 
> > It would be useful if you could specify a language in the request. This would allow community members to make tools that can use by people in any language
> 
> Unfortunately the API does not process the data, it only retrieves it from the database. This can result in you seeing items that are already localised to different languages.

Yeah, but there is no real solution of getting item stats, rarity etc. for other languages without entering every single possible translation (let alone when skyblock gets full translation support) or manually creating an entire skyblock item database based on IDs with stats, which are both very inefficient.

**Comment by nstringham** [comment]
Doesn't hypixel have a translation table somewhere that could be used?

**Comment by MartinNemi03** [comment]
> 
> 
> Doesn't hypixel have a translation table somewhere that could be used?

It doesn't afaik. Only way to get these translated strings is to be on Crowdin teams for every single language and checking for everything or manually go for them on "bugged" languages.

**Comment by FlorianWendelborn** [comment]
I’d also appreciate it if this would be fixed at some point. It makes it very hard to properly parse items for third-party skyblock integrations
